draft-ietf-ice-rfc5245bis.xml

```diff
@@ -3148,23 +2882,22 @@ stream is removed.
 
 <section anchor="sec-send-media" title="Sending Media">
 
+
 <t>
-Procedures for sending media differ for full and lite implementations.
+An agent MAY send media on any valid candidate pair before a candidate
+pair is selected.
 </t>
 
-<section title="Procedures for Full Implementations">
+<t>
+An agent MUST send media on the selected candidate pair after it is
+selected.
+</t>
 
 <t>
-Agents always send media using a candidate pair, using candidate pairs
-in the VALID LIST. Once a candidate pair has been selected only that
-candidate pair (referred to as selected pair) is used for
-sending media. An agent will send media to the remote candidate (i.e.,
-setting the destination address and port of the packet equal to that
-remote candidate), and will send it from the base associated with the
-candidate pair used for sending media. In case of a relayed
-candidate, media is sent from the agent and forwarded through
-the base (located in the TURN server), using the procedures defined
-in <xref target="RFC5766"/>.
```

Comment:
For eyeballs?  I think it's equivalent, except for the redundant part (how many times do we have to defined "selected pair"???).